iosxr_iso2vbox.py

```diff
@@ -124,10 +149,10 @@ def main(argv):
     # Handle Input ISO (Local or URI)
     if re.search(':/', args.ISO_FILE):
         # URI Image
-        cmdstring = 'scp %s@%s .' % (getpass.getuser(), args.ISO_FILE)
+        cmd_string = 'scp %s@%s .' % (getpass.getuser(), args.ISO_FILE)
```

Comment:
http://stackoverflow.com/questions/5082452/python-string-formatting-vs-format

was where I looked before and decided to change

fyi

On 7/14/16, 3:38 PM, Richard Wellum wrote:
>> @@ -124,10 +149,10 @@ def main(argv):
>>       # Handle Input ISO (Local or URI)
>>       if re.search(':/', args.ISO_FILE):
>>           # URI Image
>> -        cmdstring = 'scp %s@%s .' % (getpass.getuser(), args.ISO_FILE)
>> +        cmd_string = 'scp %s@%s .' % (getpass.getuser(), args.ISO_FILE)
>
> @glennmatthews @msherif1234 Guys any preference here?
>
> ---
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub:
> https://github.com/ios-xr/iosxrv-x64-vbox/pull/10/files/fc29a814958e64ddff6a567916527f1c4084fe76#r70869399
>